packages/chain-controller/src/ChainController.test.ts

```diff
@@ -0,0 +1,152 @@
+import { ControllerMessenger } from '@metamask/base-controller';
+import type { InternalAccount } from '@metamask/keyring-api';
+import type { SnapId } from '@metamask/snaps-sdk';
+
+import type { AllowedActions, ChainControllerActions } from './ChainController';
+import { ChainController } from './ChainController';
+
+const snapId = 'local:localhost:3000' as SnapId;
+
+const address = 'bc1qrp0yzgkf8rawkuvdlhnjfj2fnjwm0m8727kgah';
+const scope = 'bip122:000000000019d6689c085ae165831e93';
+const asset = `${scope}/asset:0`;
```

Comment:
I noticed [CAIP-2](https://chainagnostic.org/CAIPs/caip-2) has been adopted for the chain identifier. The asset variable seems to be adopting a similar definition of [CAIP-19](https://chainagnostic.org/CAIPs/caip-19), but the asset namespace was set to `asset`. [CAIP-20](https://chainagnostic.org/CAIPs/caip-20) specifies the a chain agnostic asset namespace we could use. 

We are currently discussing the adoption of these CAIPs in [this slack thread](https://consensys.slack.com/archives/C06EULX1456/p1715185176384949) and it would be very valuable to hear the rationale of the `asset` asset namespace decision or other thoughts about adopting these 🙏

Reply:
Ah yes sorry, I used `asset` here, cause it's "accepted" as a valid asset namespace. And since this is a test with a mock, the actual value is not really important (as long as it's a valid CAIP-19 identifier at least).

I will change that to use `slip44` here, so that it would also "match" the actual `scope`.

Thanks for spotting this :)